policies/versioning-policy.md

```diff
@@ -0,0 +1,133 @@
+[stable release update policy].
+
+## Minor release
+
+A minor release is indicated by changing the _second_ digit of the version.
+A minor release can, and generally will, introduce new features.  However both
+the API and ABI will be preserved.
+
+> For example, a program built with OpenSSL release 3.0.1 will be able to
+> run with OpenSSL 3.1.0 but might not be able to take advantage of new
+> features without modification.
+
+Exceptions to these rules require a vote by the OMC.
+
+## Patch release
+
+A patch release is indicated by changing the _final_ digit of the version.
+A patch release will only contain bug and security fixes.
+Both the API and ABI will remain compatible across patch releases.
+
+> For example, a program linked with OpenSSL release 3.0.0 can
+> run with OpenSSL 3.0.1 without changes.
+
+Exceptions to these rules require a vote by the OMC.
+
+## Long term stable release
+
+A release can be designated as a release.  LTS releases will be supported
+for at least five years and the project will specify a LTS release at
+least every four years.  Non-LTS releases will be supported for at least
+two years.
+
+During the final year of support, we do not commit to anything other
+than security fixes.  Before that, bug and security fixes will be applied
+as appropriate.
+
+The addition of new platforms to LTS branches is acceptable so long as
+the required changes consist solely of additions to configuration.
+
+## Pre-release versions
+
+Before a major release the project will generally make a number of pre-releases.
+These are labelled _alpha_ and _beta_ releases.
+
+### Alpha release
+
+An _alpha_ release is one that:
+
+- is not necessarily feature complete and
+- does not necessarily includes all new APIs.
+
+### Beta release
+
+A _beta_ release is one that:
+
+- is feature complete;
+- indicates a feature freeze and
+- only bug fixes are permitted.
+
+## History
+
+From release 1.0.0, but prior to release 3.0.0, the OpenSSL versioning scheme
+was different and it is detailed here for historic purposes.
+
+- Letter releases, such as 1.0.2a, exclusively contain bug and security
+  fixes and no new features.
+- Releases that change the last digit, e.g. 1.1.0 vs. 1.1.1, can and
+  are likely to contain new features, but in a way that does not break
+  binary compatibility.  This means that an application compiled and
+  dynamically linked with 1.1.0 does not need to be recompiled when the
+  shared library is updated to 1.1.1.
+- Releases that change the second digit, e.g. 1.0.0 vs 1.1.0, break
+  both ABI and API compatibility.  The primary instance of this was
+  the transition to opaque internal structures occurred with OpenSSL
+  release 1.1.0.
+
+It should be noted that some features are transparent to the application
+such as the maximum negotiated TLS version and cipher suites, performance
+improvements and so on.  There is no need to recompile applications to
+benefit from these features.
```

Comment:
I'm not really sure what the intent of this para is.

Reply:
Me either.  It's taken entirely from the description of earlier versioning in the original web page.

I can remove?

Reply:
I'd say remove it.

Reply:
Removed.

Reply:
I think you pushed this fix to the wrong branch. This wording has gone from #8 but not from this PR.

Reply:
You're correct.  #8 is based over this one and I've muddled things a few times now.